.github/workflows/buf-pull-request.yml

```diff
@@ -71,22 +67,12 @@ jobs:
           cd /tmp
           curl -sSfL -O https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-x86_64.zip
           unzip protoc-${PROTOC_VERSION}-linux-x86_64.zip -d $HOME/.local
+        env:
+          PROTOC_VERSION: "23.3"
 
-      # N.B. The freshness check can have false negatives, if `prost` output
-      # is superficially but not substantively different. That's OK for now:
-      # we're aiming to keep the defs in sync, and manual maintenance is required.
+      # We ensure that the codegen runs without error, but we don't inspect
+      # the generated output for diffs, because different versions of `protoc`
+      # can generated non-substantive changes, causing noisy CI failures.
       - name: Compile protobuf specs into rust src files
         shell: bash
-        run: |
-          ./deployments/scripts/protobuf-codegen
-          s="$(git status --porcelain)"
-          if [[ -n "$s" ]]; then
-              echo "ERROR: protobuf files must be regenerated and committed."
-              echo "Run this command locally: ./deployments/scripts/protobuf-codegen"
-              echo "Make sure you're running protoc $(cut -f1 -d. <<< ${PROTOC_VERSION}).x locally."
-              echo "These are the files that reported differences:"
-              echo "$s"
-              exit 1
-          else
-              echo "OK: no changes required to protobuf specs"
-          fi
```

Comment:
We do want to check for changes in the generated proto code, though, just not changes to the proto descriptor.

Reply:
https://github.com/penumbra-zone/penumbra/commit/c9cac571502d9da3e2548a684e2dd0c37dcc2f2c

Reply:
OK, I'll make that change. That means we'll still have failing CI for some cosmetic-only changes such as https://github.com/penumbra-zone/penumbra/pull/2739/files#diff-4c7f8084527df765ac0133b6828891e9b82fb03557bacf323960235942b56c3aR3, but those should be easily sorted with a bit of visual inspection.

Reply:
Done.